index.html

```diff
@@ -3404,6 +3397,13 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+        <p>
+The media type `application/vc+ld+json` does not imply any particular securing
```

Comment:
The language suggested in this PR is an attempt to clarify that in spite of the requirement to include a specific `@context`, and in spite of that `@context` containing references to a specific securing mechanism, no specific securing mechanism should be assumed. Just because the `@context` points to data integrity doesn't mean the data has been secured.

Happy for  further suggestions on how to better communicate that here.